File: orderm8/src/main/java/edu/htl/orderm8/Data/Objects/Product.java
package edu.htl.orderm8.Data.Objects;

import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
public class Product {
	private long idProduct;
	private long fkType;
	private String name;
	private double price;
	private long quantity;
	
	public Product() {
		
	}
	
	public Product(long idProduct, long fkType, String name, double price, long quantity) {
		super();
		this.idProduct = idProduct;
		this.fkType = fkType;
		this.name = name;
		this.price = price;
		this.quantity = quantity;
	}

	public long getIdProduct() {
		return idProduct;
	}

	public void setIdProduct(long idProduct) {
		this.idProduct = idProduct;
	}

	public long getFkType() {
		return fkType;
	}

	public void setFkType(long fkType) {
		this.fkType = fkType;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public double getPrice() {
		return price;
	}

	public void setPrice(double price) {
		this.price = price;
	}

	public long getQuantity() {
		return quantity;
	}

	public void setQuantity(long quantity) {
		this.quantity = quantity;
	}

	@Override
	public String toString() {
		return "Product [idProduct=" + idProduct + ", fkType=" + fkType + ", name=" + name + ", price=" + price
				+ ", quantity=" + quantity + "]";
	}
	
}

File: XamOrderM8_Barman/XamOrderM8_Barman/XamOrderM8_Barman/Model/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace XamOrderM8_Barman.Model
{
    public class Product
    {
        public int fkType { get; set; }
        public int idProduct { get; set; }
        public string name { get; set; }
        public double price { get; set; }
        public int quantity { get; set; }

        public Product(int fkType, int idProduct, string name, double price, int qty)
        {
            this.fkType = fkType;
            this.idProduct = idProduct;
            this.name = name;
            this.price = price;
            quantity = qty;
        }

        public override string ToString()
        {
            return $"[Product fkType={fkType}, idProduct={idProduct}, name={name}, price={price}]";
        }
    }
}
